gulp/tasks/images.js

```diff
@@ -0,0 +1,60 @@
+import webp from "gulp-webp";
+import imagemin from "gulp-imagemin";
+
+export function prepareImages(err) {
+	if(app.isBuild){
+		convertToWebp();
+		compress();
+	} else {
+		copyToDist();
+		browsersync();
+	}
+	err();
+}
+
+function convertToWebp() {
+	return app.gulp.src(app.path.src.images)
+		.pipe(app.plugins.plumber(
+			app.plugins.notify.onError({
+				title: "imagesToWebp",
+				message: "Error: <%= error.message %>"
+			})
+		))
+		.pipe(app.plugins.newer(app.path.build.images))
+		.pipe(webp())
+		.pipe(app.gulp.dest(app.path.build.images));
```

Comment:
it seems to me that you can save webp in src near usual images and then with the compress function you compress all images in src and put them to dist